README.md

```diff
@@ -1,6 +1,93 @@
-# OS2forms med Forløb installation profile for Drupal 8
+# OS2forms med Forløb installation profile for Drupal 9
 
-This is a Drupal 8 installation profile for OS2forms project.
+This is a Drupal 9 installation profile for OS2forms project.
 
 ## Usage
-It's not supposed to use this profile outside https://github.com/os2forms/os2forms8 project.
+!note It's not supposed to use this profile outside https://github.com/os2forms/os2forms8 project.
+
+
+### Create a new drupal project
+```
+composer create-project drupal/recommended-project os2forms
+```
+
+
+### Add a new repository for webform embed module to your projects composer.json
+
+(Remove the dots)
```

Comment:
Removed the lines in Readme since it is added to the composer.json already. The urls referenced are fairly self-explanatory e.g https://git.drupalcode.org/issue/webform_embed-3149360/-/tree/3149360-automated-drupal-9 . It is simply about making the module d9 compliant.